Framework/API/test/FileFinderTest.h

```diff
@@ -104,6 +104,8 @@ class FileFinderTest : public CxxTest::TestSuite {
 
     ConfigService::Instance().updateFacilities(m_facFile.path());
     ConfigService::Instance().setString("datacachesearch.directory", "");
+    const std::string propfile = ConfigService::Instance().getDirectoryOfExecutable() + "Mantid.properties";
+    ConfigService::Instance().updateConfig(propfile);
```

Comment:
I thought this could affect other tests, but they all ran fine.
We just need the user directories, everything else is ignored.
I'll just set the user directories using the api, I agree that is safer and clearer

Reply:
Oh I see, for loading the data. I know there are test data folders which hold data for unit tests specifically, maybe you can find out how those are typically used

Reply:
Well, what is happening is that there is a `Mantid.properties` file in the `bin` folder but there is also another properties file in the conda environment folder, which takes priority over the one in the bin folder. When running on Jenkins, the one in the bin folder is used and all is well. When running on my machine, the one in my conda environment is used which causes the missing data directories.

I could set the data directories at the beginning of `FileFinderTest` by hard coding them relative to the bin directory. But if the structure of the build folder changes, then the test would start failing. I also thought about reading the `Mantid.properties` file in the bin folder to look for the data directories, but since `ConfigService` is a singleton, I can't really read the properties file without also updating the properties that the singleton holds, which is what I am already doing anyway with the `updateConfig` call above (unless, of course, I change the ConfigService class a bit to allow for this).

I couldn't find any other test where the data directories are updated in the config service at the beginning of the test, the closest I got was how the config service is updated several times with `updateConfig` in the `ConfigServiceTest`, in the same way that I am doing here.

I think this is the best solution we can do, because it doesn't affect other tests and even when the test is run locally, it doesn't change the properties files after running it (because at the start of workbench the priority defaults to the properties file in the conda environment).

Reply:
Or I can just hard code the paths as well, whichever you prefer